src/main/java/racingcar/domain/RacingCar.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.domain;
+
+import racingcar.dto.RacingCarDto;
+
+public class RacingCar implements Comparable<RacingCar> {
+    private static final Integer DEFAULT_POINT = 1;
+    private final String name;
+    private Integer point;
+
+    public RacingCar(String name) {
+        this.name = name;
+        this.point = DEFAULT_POINT;
+    }
+
+    private void plusPoint() {
+        point++;
+    }
+
+    public void advance(Boolean judgementResult) {
+        if (judgementResult) {
+            plusPoint();
+        }
+    }
+
+    public String getDesc() {
+        String delimiter = " : ";
+        String description = name +
+                delimiter +
+                "-".repeat(point);
+        return description;
```

Comment:
```suggestion
        return name + delimiter + "-".repeat(point);
```
바로 return 해줘도 되겠네요 ~ 매직넘버는 상수로 빼주세요